_config.yml

```diff
@@ -7,7 +7,7 @@ relative_permalinks: false
 analytics_account: "UA-3769691-28"
 
 # build settings
-exclude: ["script", "vendor", "bower.json", "Gemfile", "Gemfile.lock", "Rakefile", "readme.md", "package.json"]
+exclude: ["script", "vendor", "bower.json", "Gemfile", "Gemfile.lock", "Rakefile", "readme.md", "package.json", "node_modules"]
```

Comment:
I'm super 👌 with this, but want to confirm that adding this into the excludes doesn't screw up the `load_paths` below (not sure how it could be loaded if excluded, but then again, I don't Jekyll, so.....)

Reply:
Hey @MrChrisW , trusting you here.

Reply:
@jbjonesjr Thanks 💜